src/plugin.js

```diff
@@ -119,6 +120,10 @@ function handleTouchStart(e) {
  * @param  {Event} e
  */
 function handleContextMenu(e) {
+  if (this.contextmenu.options.disabled) {
+    // videojs-contextmenu is disabled
+    return;
+  }
```

Comment:
This won't prevent the abstracted event (`vjs-contextmenu`) from being triggered based on "long tap" detection. It should probably move to the `sendAbstractedEvent` function.